src/Parser.ts

```diff
@@ -1,15 +1,10 @@
 /**
  * @since 1.0.0
  */
-import type { Chunk, NonEmptyChunk } from "@effect/data/Chunk"
-import type { Either } from "@effect/data/Either"
-import type { LazyArg } from "@effect/data/Function"
-import type { Option } from "@effect/data/Option"
-import type { Predicate } from "@effect/data/Predicate"
 import * as internal from "@effect/parser/internal_effect_untraced/parser"
 import type { ParserError } from "@effect/parser/ParserError"
 import type { Regex } from "@effect/parser/Regex"
-
+import type { Chunk, Either, Function, Option, Predicate } from "effect"
```

Comment:
For library code, we generally use namespaced imports directly from the module to help with tree-shaking.

```ts
import * as Effect from "effect/Effect"
// ...
```

Reply:
Will do!